fleetshard/pkg/central/cloudprovider/awsclient/rds.go

```diff
@@ -40,6 +40,7 @@ const (
 	dbBackupRetentionPeriod = 30
 	dbInstancePromotionTier = 2 // a tier of 2 (or higher) ensures that readers and writers can scale independently
 	dbCACertificateType     = "rds-ca-ecc384-g1"
+	dataplaneClusterNameKey = "DataplaneClusterID"
```

Comment:
```suggestion
	dataplaneClusterNameKey = "DataplaneClusterName"
```
There is a cluster ID and a cluster name, so better have consistent naming to avoid confusion.

Reply:
The name is human readable (e.g. `acs-prod-dp-01`), the other is a UUID returned by OCM (e.g. `21nvusc7l7r1ea90errgb8504cu7ose0`). In principle we could use both here to filter the metrics.